src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java

```diff
@@ -548,6 +578,13 @@ private List<Position> decodeBinaryE(Channel channel, SocketAddress remoteAddres
                         buf.readUnsignedByte(); // label
                         position.set(Position.PREFIX_TEMP + (id - 0x2A), buf.readShortLE() * 0.01);
                         break;
+                    case 0x4B:
+                        position.set("networkVersion", buf.readUnsignedByte());
+                        position.set("networkType", buf.readUnsignedByte());
+                        int networkDescLen = buf.readUnsignedByte();
+                        String networkDesc = buf.readSlice(networkDescLen).toString(StandardCharsets.US_ASCII);
+                        network.setRadioType(networkDesc);
```

Comment:
Again, inline.

Also, what values does it return? It doesn't seem to match the expected types.

Reply:
If you want to use here, we need to know all possible values.

Reply:
Ok, I'll ignore these parameters until I find more specifics about them.